packages/btcstaking/src/scripts_utils.rs

```diff
@@ -1,106 +1,96 @@
-use crate::error::Error;
+use crate::errors::Error;
 use crate::Result;
 use bitcoin::blockdata::script::Builder;
 use bitcoin::opcodes::all::{
     OP_CHECKSIG, OP_CHECKSIGADD, OP_CHECKSIGVERIFY, OP_CSV, OP_NUMEQUAL, OP_NUMEQUALVERIFY,
-    OP_PUSHNUM_1,
 };
 
-use bitcoin::secp256k1::PublicKey;
-use bitcoin::taproot::LeafVersion;
 use bitcoin::ScriptBuf;
-use bitcoin::{TapNodeHash, TapTweakHash, XOnlyPublicKey};
 
-use k256::elliptic_curve::sec1::ToEncodedPoint;
-use k256::elliptic_curve::subtle::Choice;
 use k256::schnorr::VerifyingKey;
-use k256::{
-    elliptic_curve::{ops::MulByGenerator, point::DecompressPoint, PrimeField},
-    AffinePoint, ProjectivePoint, Scalar,
-};
 
-const UNSPENDABLE_KEY: &str = "0250929b74c1a04954b78b4b6035e97a5e078a5a0f28ec96d547bfee9ace803ac0";
+/// assemble_multisig_script is a private helper to assemble multisig script
```

Comment:
While it's a convention in Go, we have agreed to not have the redundant `assemble_multisig_script` in Rust in https://github.com/babylonlabs-io/cosmos-bsn-contracts/issues/166#issue-3186815548 :P